config/locales/views/transactions/fr.yml

```
@@ -38,18 +56,167 @@ fr:
+      pending: En attente
+      pending_tooltip: Transaction en attente — peut changer une fois validée
+      linked_with_provider: Lié avec %{provider}
+      activity_type_tooltip: Type d'activité d'investissement
+      possible_duplicate: Doublon ?
+      potential_duplicate_tooltip: Cette transaction peut être un doublon d'une autre transaction
+      review_recommended: Vérifier
+      review_recommended_tooltip: Écart de montant important — vérification recommandée pour vérifier s'il s'agit d'un doublon
+      split: Fractionner
+      split_tooltip: Cette transaction a été fractionnée en plusieurs entrées
+      split_child_tooltip: Partie d'une transaction fractionnée
+    merge_duplicate:
```

YAMLlint (1.38.0)

[error] duplication of key "merge_duplicate" in mapping

(key-duplicates)